### ISSUE TITLE
Take sorting into account

### ISSUE THREAD

**Issue by rloutrel** [Rob--W/bookmark-container-tab#17]
The official mutli-account containers allows know to sort the container.

Could your extension be updated to use this sorting information? It seems, it is using the "container creation time".

**Comment by rloutrel** [comment]
If you are not supporting your extension, can you use your voting power there: https://github.com/mozilla/multi-account-containers/issues/2341 ?

**Comment by Rob--W** [comment]
> If you are not supporting your extension, can you use your voting power there: https://github.com/mozilla/multi-account-containers/issues/2341 ?

This issue seems unrelated to the request from your initial report.

**Comment by rloutrel** [comment]
You are right. The issue I referenced would be to do the job of your extension in the extension natively supported by Mozilla. It you deprecate your extension. I assume, they would do the sorting automatically (which is the goal of my ticket).

Voting on the other ticket, for you, would mean you do not have time to update your extension any more, but you think the functionality of your extension is a MUST-HAVE (what I think) and you expect Mozilla to implement it.

**Comment by Rob--W** [comment]
> You are right. The issue I referenced would be to do the job of your extension in the extension natively supported by Mozilla. It you deprecate your extension. I assume, they would do the sorting automatically (which is the goal of my ticket).

A simplified version of the functionality of my extension is already an optional part of the official extension. I have been involved in the development and review of that.

> Voting on the other ticket, for you, would mean you do not have time to update your extension any more,

My extension is mostly feature-complete. It doesn't take that much effort to maintain it, especially in comparison to the lot of other stuff that I am doing.

**Comment by rloutrel** [comment]
Ok. Thank you for your reply :D .
Based on the history, I was afraid, the add-on was not maintained any more. But you are right the functionality is there and there is nothing more to do (out of the sorting?)
